Src/DeUrgenta.Backpack.Api/Validators/AddBackpackItemValidator.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Threading.Tasks;
+using DeUrgenta.Backpack.Api.Commands;
+using DeUrgenta.Common.Validation;
+using DeUrgenta.Domain;
+using Microsoft.EntityFrameworkCore;
+
+namespace DeUrgenta.Backpack.Api.Validators
+{
+    public class AddBackpackItemValidator : IValidateRequest<AddBackpackItem>
+    {
+        private readonly DeUrgentaContext _context;
+        public AddBackpackItemValidator(DeUrgentaContext context)
+        {
+            _context = context;
+        }
+        public async Task<bool> IsValidAsync(AddBackpackItem request)
+        {
+            return await _context.Backpacks.AnyAsync(x => x.Id == request.BackpackId);
```

Comment:
If I am an authenticated user and I know someone's backpack Id I will be able to add/delete/update/read backpack items even if I am not a backpack contributor.
In order to check if a user is a backpack contributor see `GetBackpackContributorsValidator`. 
This check should be done in all backpack item related validators.
You can get `inspiration` on tests also.